### ISSUE TITLE
Stalemate is loss

### ISSUE THREAD

**Issue by Rechefiltr-is-Fire** [WandererXII/lishogi#750]
And we have no place where it’s noted, so newcomers are already asking why they lost the drawn game.

![7CA9CBB7-1C2E-4B22-857D-A876B0D3A1B9](https://github.com/WandererXII/lishogi/assets/101206332/48240bd2-055c-4852-8975-65a8a69488b8)

**Comment by WandererXII** [comment]
Agreed, this should be mentioned somewherer. I plan to write some 'documentation' pages to solve this - rules of shogi, move notations, sfen, etc.
